sycl/include/sycl/ext/oneapi/matrix/matrix-intel.hpp

```diff
@@ -65,19 +65,39 @@ template <typename Group, typename T, use Use, size_t Rows, size_t Cols,
           layout Layout>
 struct joint_matrix;
 
+// Differentiating between the "element type" and the "storage element type"
```

Comment:
should move tf32's definition from sycl/include/sycl/ext/oneapi/matrix/matrix-tensorcores.hpp to sycl/include/sycl/ext/oneapi/matrix/matrix-unified-utils.hpp

Reply:
yeah makes sense